kong/route.go

```diff
@@ -14,7 +14,7 @@ type Route struct {
 	Paths         []*string           `json:"paths,omitempty" yaml:"paths,omitempty"`
 	PathHandling  *string             `json:"path_handling,omitempty" yaml:"path_handling,omitempty"`
 	PreserveHost  *bool               `json:"preserve_host,omitempty" yaml:"preserve_host,omitempty"`
-	Priority      *int                `json:"priority,omitempty" yaml:"priority,omitempty"`
+	Priority      *int64              `json:"priority,omitempty" yaml:"priority,omitempty"`
```

Comment:
Isn't it technically a breaking change? It will break if someone uses it like:

```go
kong.Route{ 
    Priority: kong.Int(1),
}
```

I think it should be a minor bump at least + a release note entry, WDYT?